### DISCUSSION TITLE
Chat background

### DISCUSSION THREAD

**Discussion by rlee1990** [flyerhq/flutter_chat_ui#364] [Ideas & Requests]
It would be nice for the background to take either a color or an image.

**Comment by demchenkoalex** [comment]
I think people did that already? someone actually used an image, it was somewhere in issues

**Reply by rlee1990** [reply]
I don't see it within the GitHub code but that is fine. I have decided to go a different route all together

**Reply by demchenkoalex** [reply]
https://github.com/flyerhq/flutter_chat_ui/issues/270

**Comment by suixinerdongli** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
